scripts/nn/test/compare_backends/gen_conv2d_bwd_data.dml

```diff
@@ -0,0 +1,28 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+P = floor(($H + 2 * $pad - $R) / $stride) + 1
+Q = floor(($W + 2 * $pad - $S) / $stride) + 1
```

Comment:
Similar to updating the variable names for filter sizes, can we also update the output size variables to be the same as the rest of the library?  Specifically, `P` -> `Hout`, and `Q` -> `Wout`?  Might also help to update `H` -> `Hin`, and `W` -> `Win`.  Once again, by maintaining consistency, I think this will help others in the future using and reasoning about these tests.  This would apply to the rest of the files too.